### PR TITLE
janus: Create jobs_path directory

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@kris-lab or @njam please review

File: modules/cm/manifests/services/janus.pp [hunk 1]
```diff
@@ -59,6 +59,7 @@
     plugin_rtpb_minport              => $rtpbroadcast_minport,
     plugin_rtpb_maxport              => $rtpbroadcast_maxport,
   }
+  ->
```
Comment (tomaszdurka) [inline modules/cm/manifests/services/janus.pp]: It's actually all about:

```puppet
File [$jobs_path] -> Daemon['cm-janus_video']
```
but I think we should not go into specifics.
I could also add `require => File [$jobs_path]` (only if this file defined...).

**Comment by tomaszdurka** [comment]
Actually moved the ordering into cm-janus.

**Comment by njam** [comment]
- What if "plugin_jobs_path" is `undef` in standalone?
- Via https://github.com/cargomedia/sk/issues/3070: btw instead of `/var/lib/janus/jobs`, shouldn't it be instance-specific? So for the video installation and for the audio installation?

**Comment by tomaszdurka** [comment]
@njam Please review all together, I didn't want to revert things so ended up all together.

**Comment by njam** [comment]
- file { $plugin_jobs_path: Why group "0" (and not "janus")?
- What if "plugin_jobs_path" is `undef` in standalone?
- Via https://github.com/cargomedia/sk/issues/3070: btw instead of `/var/lib/janus/jobs`, shouldn't it be instance-specific? So for the video installation and for the audio installation?

**Comment by tomaszdurka** [comment]
- Ok
- Well if it's single instance it will be ok, if you have multiple you need to adjust it as in sepc.
- It is now per instance. One for all the plugins from single instance.

**Comment by tomaszdurka** [comment]
@njam please look

**Comment by njam** [comment]
As discussed, should use the directory managed in `janus::server::setup_dirs`?

**Comment by tomaszdurka** [comment]
Solved in #1214

**Pull request closed without merging**